dali/test/python/test_dali_tf_dataset_pipelines.py

```diff
@@ -221,21 +258,26 @@ def many_input_pipeline(def_for_dataset, device, sources, input_names):
 
 
 # Test that uses multiple Generator dataset as inputs to DALI pipeline
-def external_source_converter_multiple(start_values, input_names):
+def external_source_converter_multiple(start_values, input_names, batches):
     def to_dataset(pipeline_desc, device_str):
+        dataset_pipeline, shapes, dtypes = pipeline_desc
+
         input_datasets = {}
         with tf.device('/cpu:0'):
-            for value, name in zip(start_values, input_names):
+            for value, name, batch in zip(start_values, input_names, batches):
                 tf_type = tf.dtypes.as_dtype(value.dtype)
                 shape = value.shape
                 input_dataset = tf.data.Dataset.from_generator(
                     InfiniteSampleIterator, output_types=tf_type, output_shapes=shape, args=(value,))
+                if batch is None or batch == True:
```

Comment:
```suggestion
                if batch is None or batch:
```
?

Reply:
It can be a non-empty string, in that case I need explicit comparison

Reply:
Slapped a comment here.